src/learn/ExperienceModule.js

```diff
@@ -403,7 +402,7 @@ class ExperienceModule extends Component {
           />
         </h1>
         <div className="experience-container flex">
-          <ListGroup style={{ flexBasis: "30%" }} className="overflow">
+          <div style={{ flexBasis: "30%" }} className="overflow">
```

Comment:
Thank you for getting rid of another `react-bootstrap` dependency!

Reply:
Thank you, but it was @jayeclark who noticed it first and I was just following the lead 🙂